### DISCUSSION TITLE
Subsequent history

### DISCUSSION THREAD

**Discussion by jumpingtom1** [freelawproject/courtlistener#4590] [General]
I heard @mlissner on a podcast say that CourtListener is working on a function to show whether a case is still good law (like shepardizing)  to see if it has been overruled or questioned, etc.  Is there a timeline for when that might be available? Will the data be available on the API?

**Comment by mlissner** [comment]
Hi, we don't have a ETA for the feature at the moment, but it's in active development. Once live, yes, the data will be available by API.

**Reply by jumpingtom1** [reply]
Thanks!
